grafeas-charts/values.yaml

```diff
@@ -36,3 +36,11 @@ certificates:
       -----BEGIN RSA PRIVATE KEY-----
       ...
       -----END RSA PRIVATE KEY-----
+
+resources: {}
+#  limits:
```

Comment:
Is this the default if nothing is set?

Reply:
Yes that's showing the default and an example of how to define it if required.

Reply:
Got it, thanks! Let's maybe add a one-liner comment specifying something along the lines of:
```
# Example definition
```

Reply:
It might be best to enable limits/requests by default for best practice. Any idea on sensible minimum cpu/memory requirements for running grafeas? Guessing pretty low as its a go binary.

Reply:
I haven't measured it recently, but happy to go with what shows up on your end when bringing this up, or not setting any limits in this PR. I'll leave the choice up to you.:)

Reply:
I guess people will enable HPA/increase replicas etc depending on their traffic so as long as the defaults get things up and running it should be ok. These limits seem to do that.